Subject: [SCSI] libfc: fix RNN_ID smashing skb payload
References: fate#306857, fate#306859, bnc#551175
Acked-by: Jiri Bohac <jbohac@suse.cz>
From: Joe Eykholt <jeykholt@cisco.com>

The code that filled in the name server RNN_ID (register node name)
    request had somehow gotten a line in it from the RFT_ID code
    which copies 32 bytes of data over the relatively short payload.
    This caused some corruption and hangs.

    Simply deleted the extraneous line.

    Signed-off-by: Joe Eykholt <jeykholt@cisco.com>
    Signed-off-by: Robert Love <robert.w.love@intel.com>
    Signed-off-by: James Bottomley <James.Bottomley@suse.de>
---

 include/scsi/fc_encode.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)


diff --git a/include/scsi/fc_encode.h b/include/scsi/fc_encode.h
index 9afcbb9..c8968d3 100644
--- a/include/scsi/fc_encode.h
+++ b/include/scsi/fc_encode.h
@@ -134,7 +134,6 @@ static inline int fc_ct_fill(struct fc_lport *lport,
 		ct = fc_ct_hdr_fill(fp, op, sizeof(struct fc_ns_rn_id));
 		hton24(ct->payload.rn.fr_fid.fp_fid,
 		       fc_host_port_id(lport->host));
-		ct->payload.rft.fts = lport->fcts;
 		put_unaligned_be64(lport->wwnn, &ct->payload.rn.fr_wwn);
 		break;
 
